tftpd-hpa: fix AR definition

Make sure that $(AR) can be safely redefined for crossbuilding, by
moving its parameters into the actual make targets, so $(AR) can be
properly expanded to the cross-toolchain ar implementation.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/config/MCONFIG.in
+++ b/config/MCONFIG.in
@@ -66,5 +66,5 @@ LIBOBJS    = @LIBOBJS@
 TFTPDOBJS = @TFTPDOBJS@
 
 # ar and ranlib (for making libraries)
-AR	= @AR@ cq
+AR	= @AR@
 RANLIB	= @RANLIB@
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -22,5 +22,5 @@ install:
 
 libxtra.a: $(LIBOBJS)
 	-rm -f libxtra.a
-	$(AR) libxtra.a $(LIBOBJS)
+	$(AR) cq libxtra.a $(LIBOBJS)
 	$(RANLIB) libxtra.a
--- a/common/Makefile
+++ b/common/Makefile
@@ -17,7 +17,7 @@ all: $(LIB)
 
 $(LIB): $(OBJS)
 	-rm -f $(LIB)
-	$(AR) $(LIB) $(OBJS)
+	$(AR) cq $(LIB) $(OBJS)
 	$(RANLIB) $(LIB)
 
 $(OBJS): tftpsubs.h
